lib/bolt/node/winrm.rb

```diff
@@ -73,9 +73,24 @@ def shell_init
   $startInfo.RedirectStandardOutput = $true
   $startInfo.RedirectStandardError = $true
 
+  $stdoutHandler = { $Event.SourceEventArgs.Data | Out-Host }
+  $stderrHandler = { Write-Error $Event.SourceEventArgs.Data }
+  $invocationId = [Guid]::NewGuid().ToString()
+  $exitedHandler = { New-Event -SourceID $invocationId }
+
   try
   {
-    $process = [System.Diagnostics.Process]::Start($startInfo)
+    $process = New-Object System.Diagnostics.Process
+    $process.StartInfo = $startInfo
```

Comment:
I think `EnableRaisingEvents` needs to be set per https://msdn.microsoft.com/en-us/library/system.diagnostics.process.enableraisingevents(v=vs.110).aspx (even though local tests seem to indicate this is behaving properly)

Reply:
This is done, though GitHub hasn't noticed...